### PR TITLE
update to new docs location

### PR DESCRIPTION
**Pull request by dholbach**
These are all the mentions of docs.fluxcd.io on the website itself.

One question:
- v1: we now have <https://fluxcd.io/install> which is a download script for `fluxctl` - what's the original location for it? Do we advertise this?
- v2: <https://fluxcd.io/install.sh> is still what we advertise - it's unaffected and still working the way it should.

Just thinking having both links might be confusing - is the former used anywhere?

### CODE REVIEW

**Comment by dholbach** [comment]
:heavy_check_mark:

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ This repo houses the assets used to build the Flux project's landing page at <ht
 
 > **Note**: The sources for some of Flux's documentation are housed in the other Flux repositories within <https://github.com/fluxcd>. Issues and pull requests for this documentation should be registered at that repos.
```
Comment (stefanprodan) [inline README.md:7]:
```suggestion
> **Note**: The sources for some of Flux's documentation are housed in other repositories within <https://github.com/fluxcd>. Documentation issues and pull requests should be made against those repos.
```

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ This repo houses the assets used to build the Flux project's landing page at <ht
...
 >
-> Project       | Docs Site                                        | Github Source
-> ------------- | ------------------------------------------------ | -------------
-> Flux v1       | <https://docs.fluxcd.io>                         | <https://github.com/fluxcd/flux>
-> Helm Operator | <https://docs.fluxcd.io/projects/helm-operator/> | <https://github.com/fluxcd/helm-operator>
-> Flagger       | <https://docs.flagger.app>                       | <https://github.com/fluxcd/flagger>
-> Flux v2       | <https://fluxcd.io/docs>                         | <https://github.com/fluxcd/website>
+> Project       | Docs Site                                 | Github Source
+> ------------- | ------------------------------------------| -------------
+> Flux v1       | <https://fluxcd.io/legacy/flux>           | <https://github.com/fluxcd/website>
```
Comment (stefanprodan) [inline README.md:11]: I would list this as Flux v2, Flagger, Flux v1, Helm Operator v1.

File: README.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ This repo houses the assets used to build the Flux project's landing page at <ht
...
+> Helm Operator | <https://fluxcd.io/legacy/helm-operator/> | <https://github.com/fluxcd/website>
+> Flagger       | <https://docs.flagger.app>                | <https://github.com/fluxcd/flagger>
+> Flux v2       | <https://fluxcd.io/docs>                  | <https://github.com/fluxcd/website>
 >
-> We are in the process of moving Flux v1, Helm Operator and Flagger docs into this repository as well.
+> We are in the process of moving Flagger docs into this repository as well.
```
Comment (stefanprodan) [inline README.md:16]: We are not in the process of moving Flagger docs, maybe next year, please remove this.

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach ☕

**Pull request merged**